src/keria/app/ipexing.py

```diff
@@ -144,6 +143,8 @@ def sendMultisigExn(agent, hab, ked, sigs, atc, rec):
         if grant is None:
             raise falcon.HTTPBadRequest(description=f"attempt to admit an invalid grant {admitked['p']}")
 
+        agent.exchanges.append(dict(said=serder.said, pre=hab.pre, rec=rec, topic='credential'))
```

Comment:
I've moved this below the above checks for the exn message and link to grant etc - presume we don't want to send the `/multisig/exn` message to others until we have passed those checks